code/datums/emergency_calls/tank_crew.dm

```diff
@@ -24,11 +24,10 @@
 	sleep(5)
 	arm_equipment(H, /datum/equipment_preset/uscm/tank/full, TRUE, TRUE)
 	to_chat(H, SPAN_ROLE_HEADER("You are a Vehicle Crewman in the USCM"))
-	to_chat(H, SPAN_ROLE_BODY("You are here to assist in the defence of the [SSmapping.configs[GROUND_MAP].map_name]. Listen to the chain of command."))
+	to_chat(H, SPAN_ROLE_BODY("You are here to assist in the defence of [SSmapping.configs[GROUND_MAP].map_name]. Listen to the chain of command."))
 	to_chat(H, SPAN_BOLDWARNING("If you wish to cryo or ghost upon spawning in, you must ahelp and inform staff so you can be replaced."))
 
 	sleep(10)
 	to_chat(H, SPAN_BOLD("Objectives: [objectives]"))
 
 	GLOB.data_core.manifest_inject(H) //Put people in crew manifest
-
```

Comment:
The double-newlines like this getting removed in some files is because of my VSCode style settings I think. I can revert them if that would be preferred.
(To clarify, there's still one empty line at the end of the file, just not any more than that.)